VBA/Excel-VBA/articles/excel-performance-and-limit-improvements.md

```diff
@@ -1,247 +1,165 @@
+
+### Full column references
+Previously, workbooks using large numbers of full column references and multiple worksheets (for example `=COUNTIF(Sheet2!A:A,Sheet3!A1)`), might use large amounts of CPU and memory when opened, or rows were deleted. An improvement in Excel 2016 Build 16.0.8212.1000 substantially reduces the memory and CPU used in these circumstances.
  
-*My test on a workbook with 6 million formulas using full column references failed with an Out of Memory message at 4 GB of virtual memory with Excel 2013 LAA but only used 2 GB virtual memory with Excel 2016*.
+*My test on a workbook with 6 million formulas using full column references failed with an Out of Memory message at 4 GB of virtual memory with Excel 2013 LAA, but only used 2 GB virtual memory with Excel 2016*.
 
-### Structured References
-In some circumstances editing Excel Tables where formulas in the workbook use Structured References to the Table could be slow with Excel 2013 and previous versions. This led to the perception that Tables should not be used with large numbers of rows.
-Excel 2016 has now fixed this problem. 
+### Structured references
+In some circumstances, editing Excel tables where formulas in the workbook use structured references to the table could be slow with Excel 2013 and previous versions. This led to the perception that tables should not be used with large numbers of rows. Excel 2016 has now fixed this problem. 
 
 *My test showed an editing operation that took 1.9 seconds in Excel 2013 took about 2 milliseconds in Excel 2016.*
 
-For more details see [Why Structured References are Slow in Excel 2013 but fast in Excel 2016](https://fastexcel.wordpress.com/2017/02/19/why-structured-references-are-slow-in-excel-2013-but-fast-in-excel-2016/).
-
-### Filtering, Sorting Copy/Pasting
-The Excel 2016 team studied a number of large workbooks that show slow response when using Filtering, Sorting and Copy/Pasting, and a number of improvements have been made:
+For more information, see [Why Structured References are Slow in Excel 2013 but fast in Excel 2016](https://fastexcel.wordpress.com/2017/02/19/why-structured-references-are-slow-in-excel-2013-but-fast-in-excel-2016/).
 
-In Excel 2013 after Filtering or Sorting or Copy/Pasting many rows Excel could be slow responding or would hang. Performance was highly dependent on the count of all rows between the top visible row and the bottom visible row. An improvement made to the internal calculation of vertical user interface positions in build 16.0.8431.2058 has made these operations much faster. 
+### Filtering, sorting, copy/pasting
+The Excel 2016 team studied a number of large workbooks that show slow response when using filtering, sorting, and copy/pasting, and a number of improvements have been made.
```

Comment:
Revise as "We've made a number of improvements to the response time when filtering, sorting, and copy/pasting in large workbooks."